vendor.conf

```diff
@@ -1,7 +1,7 @@
 github.com/beorn7/perks                             4c0e84591b9aa9e6dcfdf3e020114cd81f89d5f9
 github.com/BurntSushi/toml                          3012a1dbe2e4bd1391d42b32f0577cb7bbc7f005 # v0.3.1
 github.com/containerd/btrfs                         af5082808c833de0e79c1e72eea9fea239364877
-github.com/containerd/cgroups                       fada802a7909d430bd17126fd6e4bbf5716f2bcd
+github.com/containerd/cgroups                       06e7180859011d9f463faa337f661bd8c97139c2
```

Comment:
can we vendor the latest revision?

Reply:
I think we don't need this until we will fix `vndr` issue?